Log/SentryLogger.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Bread\BreadCheckout\Log;
+
+/**
+ * Class SentryLogger
+ * @package Bread\BreadCheckout\Log
+ */
+class SentryLogger {
+
+    public static function init() {
+        try {
+            \Sentry\init([
+                'dsn' => 'https://a832bffb9c574aaa9e4a113b61f90e9e@sentry.io/1263830',
```

Comment:
Should dns string be exposed like this, meaning if we distribute extension with same dns string to each merchant, and our git repository is public, does that create some security concern?

Reply:
yeah we're going to be setting up a lambda function to provide that, that's still something that's outstanding that I'm working on!